### ISSUE TITLE
🐛 Bug: Ersilia complains about packaging module

### ISSUE THREAD

**Issue by DhanshreeA** [ersilia-os/ersilia#1409]
### Describe the bug.

Refer to the logs:

```bash
22:38:26 | DEBUG    | Repo path specified: models-test/eos3b5e
22:38:26 | DEBUG    | Absolute path: /home/dee/ersilia-project/models-test/eos3b5e
⬇️  Fetching model eos3b5e: molecular-weight
22:38:26 | DEBUG    | Initialized with URL: None
22:38:26 | DEBUG    | Getting model source
22:38:26 | DEBUG    | Model getting fetched from Local path
22:38:26 | DEBUG    | Starting fetching procedure
22:38:26 | DEBUG    | Fetching in your system, not from DockerHub
22:38:26 | DEBUG    | Model doesn't exist in your local, fetching it now
22:38:26 | DEBUG    | Deciding fetcher (BentoML or FastAPI)
22:38:26 | DEBUG    | Fetching using BentoML
🚨🚨🚨 Something went wrong with Ersilia 🚨🚨🚨

Error message:

No module named 'packaging'
If this error message is not helpful, open an issue at:
 - https://github.com/ersilia-os/ersilia
Or feel free to reach out to us at:
 - hello[at]ersilia.io

If you haven't, try to run your command in verbose mode (-v in the CLI)
 - You will find the console log file in: /home/dee/eos/current.log
 ```

Technically packaging should ship natively with Python, so this shouldn't be happening. I tested this with both Python 3.10.14, and with 3.10.15. 

### Operating environment

Ubuntu 24.04 LTS (GitHub Actions)

**Comment by GemmaTuron** [comment]
Hi @DhanshreeA 

This is very concerning, is it an error that happens to any user? with which version of Ersilia?

**Comment by DhanshreeA** [comment]
This would happen to any user or really in any new environment created for Ersilia. It seems that one of the new dependencies we have added for all the async stuff might be the culprit. I think this issue would happen 0.1.38 onwards since these dependencies weren't there in the 0.1.37 or its predecessors.

**Comment by DhanshreeA** [comment]
@Abellegese we need to look into this by inspecting the new dependencies.

**Comment by Abellegese** [comment]
Yes definitely @DhanshreeA its probabily one of the async packages. I am taking a look into it.

**Comment by Abellegese** [comment]
Hey @DhanshreeA I think the problem is `bentoml` let me explain. When ersilia get installed bentoml will not be installed automatically. So apparently `packaging` module will be installed as `bentoml` dependency. Naturally `bentoml` is configured to be installed when its command is called anywhere in the code or in the terminal(ersilia installed in it). I called `bentoml` command and it automatically resolved the issue. What do you think?

**Comment by Abellegese** [comment]
When the fetcher decides to install using `bentoml` but its was not there which leads to same packing module problem. Maybe we need to somehow call this `bentoml` when `ersilia` module loads just like you did before. It won't be a problem now since you resolved the loop issue.

**Comment by DhanshreeA** [comment]
@Abellegese I hear you but I am still not convinced, because of two reasons: 
1. `packaging` is a Python native module, so ideally it should not run into such a dependency conflict to begin with.
2. Not all ersilia environments always need bentoml, for example, any new environment that fetches a model using Docker would never require bentoml. For example, look at this [commit](https://github.com/ersilia-os/ersilia/commit/e0295db3b80849dbaac3351e7905f0fabd90c37f) on the Circle CI file from 3 weeks ago. I had to manually add packaging module here because it was failing, and it did not fail before this. So it's definitely more recent changes, instead of the bentoml dependency that has been around since the beginning of Ersilia.

**Comment by DhanshreeA** [comment]
> Sorry what do you mean by async stuff? And if this happens 0.1.38 onwards it means that the Hub is actually not working? How did you solve it?

You are right @GemmaTuron those releases on any fresh environment would likely cause issues as well unless one manually installs packaging. This is not ideal, and we will address this.

**Comment by Abellegese** [comment]
Oka @DhanshreeA
